lightly/loss/hypersphere_loss.py

```diff
@@ -0,0 +1,96 @@
+"""
+https://arxiv.org/pdf/2005.10242.pdf
+
+NOTE: hypersphere is perhaps bad naming as I am not sure it is the essence;
+alignment-and-uniformity loss perhaps? Does not sound as nice.
+"""
+
+import torch
+
+
+def normalize_l2(x, dim=-1):
```

Comment:
Can we use the PyTorch functional `normalize` for this?
(see https://pytorch.org/docs/1.1.0/nn.html#normalize)

```python
import torch.nn.functional as F
x_normalized = F.normalize(x)
```

Reply:
Seems like it. Implementing it myself is a reflex of numpy refusing to do so for the part 10 years ive been using it I suppose.